docs/backend-api/resources/field_service/task/checkpoint.md

```diff
@@ -94,67 +144,167 @@ Delete checkpoint with the specified id.
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/task/checkpoint/list?hash=a6aa75587e5c59c32d347da438505fc3&checkpoint_ids=[1,2]
+    ```
 
 #### response
 
 ```json
 {
     "success": true,
-    "list": [ <checkpoint>, ... ]
+    "list": [{
+         "id": 111,
+         "user_id": 3,
+         "tracker_id": 22,
+         "location": {
+             "lat": 56.5,
+             "lng": 60.5,
+             "address": "Fichtenstrasse 11",
+             "radius": 150
+         },
+         "label": "Deliver parcels",
+         "description": "Quickly",
+         "creation_date": "2014-01-02 03:04:05",
+         "from": "2014-02-03 04:05:06",
+         "to": "2014-03-04 05:06:07",
+         "external_id": null,
+         "status": "assigned",
+         "status_change_date": "2014-01-02 03:04:05",
+         "max_delay" : 5,
+         "min_stay_duration": 0,
+         "arrival_date": "2014-01-02 03:04:05",
+         "stay_duration": 0,
+         "origin": "imported",
+         "tags": [1, 2],
+         "type": "checkpoint"
+    }]
 }
 ```
 
 #### errors
 
 [General](../../../getting-started.md#error-codes) types only.
 
-
-
 ### read
 
-Get route checkpoint by id.
+Gets route checkpoint by specified id.
 
 #### parameters
 
-* **checkpoint_id** - (int) ID of the checkpoint
+| name | description | type | 
+| :--- | :--- | :--- |
+| checkpoint_id | ID of the checkpoint. | int |
+
+#### examples
+
+=== "cURL"
+
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/task/checkpoint/read' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "checkpoint_id": 111}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/task/checkpoint/read?hash=a6aa75587e5c59c32d347da438505fc3&checkpoint_id=111
+    ```
 
 #### response
 
 ```json
 {
     "success": true,
-    "value":  ${checkpoint} // JSON object
+    "value":  {
+          "id": 111,
+          "user_id": 3,
+          "tracker_id": 22,
+          "location": {
+              "lat": 56.5,
+              "lng": 60.5,
+              "address": "Fichtenstrasse 11",
+              "radius": 150
+          },
+          "label": "Deliver parcels",
+          "description": "Quickly",
+          "creation_date": "2014-01-02 03:04:05",
+          "from": "2014-02-03 04:05:06",
+          "to": "2014-03-04 05:06:07",
+          "external_id": null,
+          "status": "assigned",
+          "status_change_date": "2014-01-02 03:04:05",
+          "max_delay" : 5,
+          "min_stay_duration": 0,
+          "arrival_date": "2014-01-02 03:04:05",
+          "stay_duration": 0,
+          "origin": "imported",
+          "tags": [1, 2],
+          "type": "checkpoint",
+          "form": <form_object>
+    }
 }
 ```
 
-where **checkpoint** described [here](#checkpoint).
+* `value` - `checkpoint` object described [here](#checkpoint object).

```

Comment:
Whitespace in the link.

Reply:
links with spaces really work with whitespaces, but I changed spaces to `-`